.ops/ecamp3-logging/templates/fluentd/fluentd_clusterfilter.yaml

```diff
@@ -0,0 +1,50 @@
+# from: https://stackoverflow.com/questions/41991128/parsing-inner-json-inside-fluentd
+apiVersion: fluentd.fluent.io/v1alpha1
+kind: ClusterFilter
+metadata:
+  name: cluster-filters
+  labels:
+    filter.fluentd.fluent.io/enabled: "true"
+    {{- include "app.commonLabels" . | nindent 4 }}
+    {{- include "app.commonSelectorLabels" . | nindent 4 }}
+spec:
+  filters:
+    - customPlugin:
+        config: |
+          <filter **>
+            @type parser
+            key_name log
+            reserve_data true
+            remove_key_name_field false
+            hash_value_field json
+            <parse>
+              @type multi_format
+              <pattern>
+                format json
+              </pattern>
+              <pattern>
+                format none
+              </pattern>
+            </parse>
+          </filter>
+          <filter **>
+            @type record_transformer
+            enable_ruby
+            <record>
+              escapedUrl ${if record['json'] && record['json']['httpRequest'] && record['json']['httpRequest']['requestUrl']; then record['json']['httpRequest']['requestUrl']; else nil; end}
```

Comment:
Should we use `requestUrl` instead of `escapedUrl` here? This would make filtering easier, if we later want to add/support log formats other than the nginx-ingress format.

Reply:
currently it is a the target variable which will be modified later in this chain to escape the ids.
I can also pull up the requestUrl, that it is a t the top level.